api/src/routes/admin.routes.ts

```diff
@@ -0,0 +1,20 @@
+import Router from 'express';
+import {adminController} from '../controllers';
+
+const adminRoutes = Router();
+adminRoutes.post('/addban', (request, response) =>
+  adminController.addBan(request, response));
+adminRoutes.post('/adddonor', (request, response) =>
```

Comment:
The RESTful way of doing these routes would be to use appropriate HTTP verbs on the same resource i.e.
```typescript
POST for creating a resource
adminRoutes.post('/bans', (request, response) => adminController.addBan(request, response));

// DELETE for destroying the ban with the given id
adminRoutes.delete('/bans/:id', (request, response) => adminController.deleteBan(request, response));
```

Reply:
Will do

Reply:
I'm having issues getting Routes.get to function properly with my usersearch, as if the search is blank, it messes up the url.  I will need to add a conditional to the method to change the endpoint url.  Is this the best way, or is there a better solution?